optuna/integration/botorch.py

```diff
@@ -63,6 +63,100 @@ def _get_sobol_qmc_normal_sampler(num_samples: int) -> SobolQMCNormalSampler:
 
 _logger = logging.get_logger(__name__)
 
+with try_import() as _imports_logei:
+    from botorch.acquisition.analytic import LogExpectedImprovement
+
+
+@experimental_func("3.3.0")
+def logei_candidates_func(
+    train_x: "torch.Tensor",
+    train_obj: "torch.Tensor",
+    train_con: Optional["torch.Tensor"],
+    bounds: "torch.Tensor",
+    pending_x: Optional["torch.Tensor"],
+) -> "torch.Tensor":
+    """Log Expected Improvement (LogEI).
+
+    The default value of ``candidates_func`` in :class:`~optuna.integration.BoTorchSampler`
+    with single-objective optimization for non-constrained problems.
```

Comment:
We can update the description of the default `candidates_func` in the `BoTorchSampler` reference as well.

https://github.com/optuna/optuna/blob/2b6addc365003eaab9e20bacb964ff15ed057da7/optuna/integration/botorch.py#L462-L468

Reply:
Thanks for your update.